x-pack/plugins/security_solution/public/management/components/artifact_card_grid/components/grid_header.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import styled from 'styled-components';
+import { CardCompressedHeaderLayout, CardSectionPanel } from '../../artifact_entry_card';
+
+const GridHeaderContainer = styled(CardSectionPanel)`
+  padding-bottom: ${({ theme }) => theme.eui.paddingSizes.s};
+`;
+
+export const GridHeader = memo(() => {
+  return (
+    <GridHeaderContainer>
+      <CardCompressedHeaderLayout
+        expanded={false}
+        expandToggle={<div style={{ width: '24px' }} />}
```

Comment:
yeah, that was my first approach, but there is no theme property for getting the width of an Icon button. The closest is the EuiIcon `l` size: 
https://gist.github.com/paul-tavares/303d718ac7f514d6ee681a3851407a7e#file-eui_theme_style_props-js-L211

But even that will not guarantee that this will provide the exact spacing we need in order for things to line up.